monai/utils/misc.py

```diff
@@ -217,3 +218,39 @@ def set_determinism(
         torch.backends.cudnn.benchmark = False
     else:
         torch.backends.cudnn.deterministic = False
+
+
+def list_to_dict(items):
+    """
+    To convert a list of "key=value" pairs into a dictionary.
+    For examples: items: `["a=1", "b=2", "c=3"]`, return: {"a": "1", "b": "2", "c": "3"}.
+    If no "=" in the pair, use None as the value, for example: ["a"], return: {"a": None}.
+
+    """
+
+    def _parse_var(s):
+        items = s.split("=")
+        # we remove blanks around keys, as is logical
+        key = items[0].strip()
+        value = None
+        if len(items) > 1:
```

Comment:
this could be implemented with `split("=", maxsplit=1)`? https://docs.python.org/3/library/stdtypes.html#str.split

should we detection the duplicated keys such as: `["a=1", "b=2", "a=3"]`?

Reply:
updated in the latest commit.
Thanks.